modular_zubbers/code/modules/clothing/head/wig.dm

```diff
@@ -0,0 +1,73 @@
+/obj/item/clothing/head/wig
+	var/obj/item/clothing/head/head_accessory
+	var/mutable_appearance/head_accessory_MA
+
+	var/overlay_on = FALSE
+
+	var/mob/living/carbon/holder
+	var/item_path
+
+/obj/item/clothing/head/wig/attackby(obj/item/I, mob/living/user)
+	. = ..()
+	//Checks if the item is in the list of items available
+	for(var/type in HEAD_ACCESSORIES_PATHS)	
+		if(istype(I, type))
+			item_path = type
+			break
```

Comment:
This is fine but if you want you do something like this too
```
if(!is_path_in_list(I.type, HEAD_ACCESSORIES_PATHS)
    break
item_path = I.type
```

Reply:
Actually I'm unsure how this handles subtypes!

Reply:
Yeahh.. Don't think there's a way of doing that without iterations